### PR TITLE
tidb-cloud: add docs learning path

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Add TiDB Cloud docs learning path

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs/pull/9560

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/learning-path.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+title: Learning Path
+hide_title: true
+hide_sidebar: true
+---
+
+<!--
+注意frontmatter里的内容，hide了右边栏和文章的title。
```
Comment (lilin90) [inline tidb-cloud/learning-path.md]: Would you please translate **all** the Chinese text in this file to English? Let's use English in this repo. Thanks!

File: tidb-cloud/learning-path.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+-->
+
+<LearningPathContainer>
+
+<!-- Tab 标签
+<LearningPathContent id="dba" label="dba title">
+-->
+<LearningPath>
+
+![avator](https://avatars.githubusercontent.com/u/773853)
```
Comment (lilin90) [inline tidb-cloud/learning-path.md:23]: Will you replace this icon with the real one in this PR?

File: tidb-cloud/learning-path.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
+---
+title: Learning Path
+hide_title: true
+hide_sidebar: true
+---
+
+<!--
+注意 frontmatter 里的内容
```
Comment (lilin90) [inline tidb-cloud/learning-path.md:7]: Would you please translate all the Chinese text in this file to English? Let's use English in this repo. Thanks!

File: tidb-cloud/learning-path.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+</LearningPath>
+
+<LearningPath>
+
+![avator](https://avatars.githubusercontent.com/u/773853)
+
+<h4>Try</h4>
+
+[Quick Start](/tidb-cloud/tidb-cloud-quickstart.md)
+[Perform a POC](/tidb-cloud/tidb-cloud-poc.md)
```
Comment (lilin90) [inline tidb-cloud/learning-path.md:61]:
```suggestion
[Perform a PoC](/tidb-cloud/tidb-cloud-poc.md)
```

File: tidb-cloud/learning-path.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+[What is TiDB Cloud](/tidb-cloud/tidb-cloud-intro.md)
+[Architecture](/tidb-cloud/tidb-cloud-intro.md#architecture)
+[High Availability](/tidb-cloud/high-availability-with-multi-az.md)
+[FAQs](/tidb-cloud/tidb-cloud-faq.md)
+
+</LearningPath>
+
+<LearningPath>
+
+![avator](https://avatars.githubusercontent.com/u/773853)
```
Comment (lilin90) [inline tidb-cloud/learning-path.md:35]: Is the "avator" word a typo? **avatar**?

File: tidb-cloud/learning-path.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+<h4>Monitor</h4>
+
+[Overview](/tidb-cloud/monitor-tidb-cluster.md)
+[Built-in Alerting](/tidb-cloud/monitor-built-in-alerting.md)
+[Datadog](tidb-cloud/monitor-datadog-integration.md)
+[Prometheus and Grafana](/tidb-cloud/monitor-prometheus-and-grafana-integration.md)
```
Comment (lilin90) [inline tidb-cloud/learning-path.md:115]: Duplicate?

**Comment by hfxsd** [comment]
Replaced by a new PR https://github.com/pingcap/docs/pull/9560

**Pull request closed without merging**